SUMMARY.md

```diff
@@ -49,6 +49,7 @@
   * [Exchange rates](operations/exchangerates.md)
   * [Images](operations/images.md)
   * [Languages](operations/languages.md)
+  * [Loyalty Programs](operations/loyaltyprograms.md)
```

Comment:
Sorry to be annoying, just a small point, I didn't notice it before; to fit in with all the other pages, it should be "Loyalty programs" instead of "Loyalty Programs"